pants.ci.toml

```diff
@@ -23,8 +23,8 @@ report = ["raw", "xml"]
 from_env_var = "TOOLCHAIN_AUTH_TOKEN"
 org = "pantsbuild"
 ci_env_variables = [
-  "TRAVIS", "TRAVIS_JOB_ID", "TRAVIS_BUILD_ID", "TRAVIS_PULL_REQUEST", "TRAVIS_BUILD_WEB_URL",
   "GITHUB_ACTIONS", "GITHUB_RUN_ID", "GITHUB_REF", "GITHUB_EVENT_NAME", "GITHUB_SHA", "GITHUB_REPOSITORY",
+  "GITHUB_WORKFLOW","GITHUB_JOB",  # temporary, for debugging issues w/ restricted tokens.
```

Comment:
Stale?

Reply:
no, I need it around for a while (for other people's PRs too) to figure out why pants is requesting so many tokens (and from where....)